cmd/run_controller/Dockerfile

```diff
@@ -4,13 +4,13 @@ RUN mkdir /build
 ADD . /build/
 WORKDIR /build
 RUN apk add --no-cache git
-RUN CGO_ENABLED=0 GOOS=linux go build -mod=readonly -a -installsuffix cgo -ldflags '-extldflags "-static"' -o main -v ./cmd/run_controller
+RUN CGO_ENABLED=0 GOOS=linux go build -mod=readonly -a -installsuffix cgo -ldflags '-extldflags "-static"' -o runcontroller -v ./cmd/run_controller
```

Comment:
A better name would be `steward-runctl`.

Reply:
Adjusted to `steward-runctl`, `steward-tenantctl`